packages/node_modules/betsy/src/index.ts

```diff
@@ -24,6 +24,22 @@ export class EventEmitter<T> {
     }
   }
 
+  emitAsync<K extends keyof T>(event: K, msg: T[K]) {
+    const listeners = this.events.get(event) || []
+
+    setTimeout(() => {
```

Comment:
requestIdleCallback maybe?

Reply:
Actually I do not think that should be Betsy responsibility. Cause pushing out the event is not what takes time, it is what you do with that information. So if you are listening to an event to do work, you should rather as the consumer use `requestIdleCallback` for efficiency... I think :)